client/app/bundles/shared/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,51 @@
+@import "../base.scss";
+
+body {
+  font-family: $font-family;
+}
+
+.fill select{
+  background:  #6D0839 url('./down.svg') no-repeat right;
+
+  @include selectDropdown(
+    $padding: 13px 32px,
+    $font-size: $font18
+  );
+  &:focus {
+    outline: none;
+  }
+}
+
+.ghost select{
+  background:  #9a9999 url('./down.svg') no-repeat right;
+
+  @include selectDropdown(
+    $padding: 16px 30px,
+    $font-size: $font22
+  );
+  &:focus {
+    outline: none;
+  }
+}
+
+.ghostsmall select {
```

Comment:
Maybe call this .ghost_small, to be consistent with how we've given class names.

Reply:
@julianguyen this has been addressed.